projects/control-service/projects/job-builder/Dockerfile

```
@@ -4,19 +4,22 @@ FROM gcr.io/kaniko-project/executor
 
 FROM alpine
 
-COPY --from=0 /kaniko /kaniko
-
+COPY --chown=1000:2000 --from=0 /kaniko /kaniko
 
 ENV PATH $PATH:/kaniko
 ENV SSL_CERT_DIR=/kaniko/ssl/certs
 ENV DOCKER_CONFIG /kaniko/.docker/
 
 WORKDIR /workspace
 
-COPY Dockerfile.python.vdk /workspace/Dockerfile
-COPY build_image.sh /build_image.sh
-RUN chmod +x /build_image.sh
+RUN cd .. \
+    && chown -R 1000:2000 workspace \
```

You can use full path to folder
```suggestion
    RUN chown -R 1000:2000 /workspace
```

---

So you don't need `cd ..` and `cd workspace`.

---

Good point. Refactored, thank you